dcc-portal-ui/app/scripts/app/js/app.js

```diff
@@ -588,15 +591,8 @@
         }
       });
 
-    // All else redirect to home
-    $urlRouterProvider.otherwise(function($injector, $location) {
-
-      $injector.invoke(['Notify', 'Page', function(Notify, Page) {
-        Page.setPage('error');
-        Notify.setMessage('Cannot find:' + ' ' + $location.url());
-        Notify.showErrors();
-      }]);
-    });
+    // If state doesn't exist, redirect to 404
+    $urlRouterProvider.otherwise('/404');
```

Comment:
The `$stateNotFound` only occurs if we are trying to transition to a state that doesn't exist. However `$urlRouterProvider` will get triggered when `invalid route is requested`